Adding a preliminary index for /ui (#9112)
flutter/website: src/ui/index.md

> title: User interface

Reviewer: How about something a little more specific for search results?

```suggestion
title: Building user interfaces with Flutter
```

short-title: UI

> description: Content covering user interface development in Flutter apps.

Reviewer: ```suggestion
description: Introduction to user interface development in Flutter.
```

which computes and describes the geometry of the widget.

## Basic widgets

Flutter comes with a suite of powerful basic widgets,
of which the following are commonly used:

**[`Text`][]**
: The `Text` widget lets you create a run of styled text
  within your application.

**[`Row`][], [`Column`][]**
: These flex widgets let you create flexible layouts in
  both the horizontal (`Row`) and vertical (`Column`) directions.
  The design of these objects is based on the web's
  flexbox layout model.

**[`Stack`][]**
: Instead of being linearly oriented (either horizontally or vertically),
  a `Stack` widget lets you place widgets on top of each other in paint order.
  You can then use the [`Positioned`][] widget on children of a
  `Stack` to position them relative to the top, right, bottom,
  or left edge of the stack. Stacks are based on the web's
  absolute positioning layout model.

**[`Container`][]**
: The `Container` widget lets you create a rectangular visual element.
  A container can be decorated with a [`BoxDecoration`][], such as a
  background, a border, or a shadow. A `Container` can also have margins,
  padding, and constraints applied to its size. In addition, a

>   `Container` can be transformed in three dimensional space using a matrix.

Reviewer: ```suggestion
  `Container` can be transformed in three-dimensional space using a matrix.
```
